OxfordBodleian/Juel-Jensen/BDLaethe30.xml

```diff
@@ -0,0 +1,865 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0"
+    xml:id="BDLaethe30"
+     xml:lang="en"
+     type="mss">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="title1">ዳዊት፡</title>
```

Comment:
Do we need to be concerned about the presence of titles in Ethiopic in light of our discussion or can these just be removed in the pdf creation phase?

Reply:
I have systematically added the title in fidal as it is a good encoding practice in BM. I think it is enough to ask the system to systematically ignore that `<title xml:lang="gez"/>` string while generating the PDF.